examples/llamaindex/retrievers-with-fga/package.json

```diff
@@ -10,11 +10,12 @@
   },
   "license": "Apache-2.0",
   "scripts": {
-    "dev": "npx tsx index.ts",
+    "start": "npx tsx index.ts",
```

Comment:
Why **start** rather than **dev**?

Reply:
Felt more natural as in start app, also no need to `npm run start` for start since it's a predefined task like test just `npm start` is enough, meant for starting apps.